jupyter_ydoc/ydoc.py

```diff
@@ -320,6 +320,8 @@ def create_ycell(self, value: Dict[str, Any]) -> Y.YMap:
         if "id" not in cell:
             cell["id"] = str(uuid4())
         cell_type = cell["cell_type"]
+        cell_source = cell["source"]
+        cell["source"] = "".join(cell_source) if isinstance(cell_source, list) else cell_source
```

Comment:
Maybe:
```suggestion
        cell_source = "".join(cell_source) if isinstance(cell_source, list) else cell_source
```
And use `cell_source` on the RHS below.